pkg/processor/processor.go

```diff
@@ -130,20 +130,24 @@ func (p *CheckProcessor) ProcessMessage(m string) error {
 	}
 
 	p.logger.Trace("Processing vulnerabilities...")
+	p.logger.Debugf("%d vulnerabilities", len(vulns))
 	sourceFindings, err := p.processVulns(vulns)
 	if err != nil {
 		return err
 	}
+	p.logger.Debugf("%d source findings", len(sourceFindings))
 
 	p.logger.Trace("Processing source...")
 	source := p.sourceFromCheck(*check, target.ID, checkTime)
-	err = p.store.ProcessSourceExecution(source, sourceFindings)
+	p.logger.Debugf("source: %#v", source)
+	p.logger.Debugf("sourceFindings: %#v", sourceFindings)
```

Comment:
Logging in l.138 is kind of repeated here at l.143. Also, do you think it's worth logging all source findings? This will produce huge output considering all the checks we parse, and in the end we log the check id and have the report stored. Let me know what were your thoughts about this.

I think it might be better to join these log traces altogether in one logging it after the call to `p.sourceFromCheck` and having a format like: `Processing source: %#v`. Let me know what do you think.

Reply:
I added the log entries to debug the issue we found about the sources with empty IDs. I don't mind removing them but as far as they're just set in `DEBUG` log level I thought they could be helpful in the future while debugging other things. Do you prefer to re-arrange and keep them or just to remove them?

Reply:
I think we can rearrange them a little bit if you like the idea.

Reply:
OK, let's do it.

Reply:
Done together